### ISSUE TITLE
 Third-party program Overwolf continues to work after a wave of bans

### ISSUE THREAD

**Issue by HoshiSasha** [ValveSoftware/Dota2-Gameplay#21103]
### Description

I met the player and started playing with him. He told me that the program works and has protection and he has already been playing with it for half a year. With it, he can see the percentage of hero bans and the heroes that other players are playing. I ask you to look into it and ban all players who use third-party programs.

### Example Match ID (and possibly Timestamp)

1063097245 this is id accaunt

**Comment by ghostdlr** [comment]
Overwolf is allowed. You can see statistics of the players that have public profiles on dotabuff.

**Comment by HoshiSasha** [comment]
> Overwolf is allowed. You can see statistics of the players that have public profiles on dotabuff.

This program sees which heroes you play, even if you specify in the settings that your statistics will not be available to third-party sites such as dotabuff and others.

**Comment by ghostdlr** [comment]
You are confusing things. 

Maybe you try it first before posting here:
https://www.overwolf.com/app/Overwolf-DotaPlus

**Comment by HoshiSasha** [comment]
> You are confusing things. There was no wave of bans for overwolf users ever. Overwolf is allowed. Only thing valve did was to hide player profiles during the pick phase. After that all statistics are visible.
> 
> Maybe you try it first before complaining: https://www.overwolf.com/app/Overwolf-DotaPlus


no, I want to play honestly and have them play against me fairly. I know for sure that it gives an advantage.

**Comment by ghostdlr** [comment]
Try it first and tell me what advatange it gives.

**Comment by HoshiSasha** [comment]
> Try it first and tell me what advatange it gives.

what you say about umbrella?
